examples/apps/mocks/function-test.yaml

```diff
@@ -0,0 +1,86 @@
+apiVersion: serverless.kyma-project.io/v1alpha1
```

Comment:
This is also a very SAP focused example. Most probably should not stay in the examples